.github/ISSUE_TEMPLATE/hotfix-release.md

```diff
@@ -0,0 +1,44 @@
+---
+name: k6 Hotfix release
+about: Bugs or security issues have been identified and they requires a patch release.
+title: 'k6 hotfix release v<major>.<minor>.<patch>'
+labels: ["release"]
+---
+
+**Release Date**:
+
+25th May 2024 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
+
+## Release Activities
+
+Note, the assumption is a minor branch release already exists from the latest minor release. This branch is named in the format `release-v<major>.<minor>.x`.
+
+#### Documentation
+
+- [ ] If required, open and merge pull-request from `main` applying the changes to the affected version.
+
+#### In k6 repository, if the defect affects the latest release and the `master` branch
+
+- [ ] Open, get approvals and merge on `master` the patch for the defect.
```

Comment:
I believe there is a "gray zone" when the defect affects the master, but `master` contains the changes in the affected logic.

Reply:
In that case, we need to patch each version independently. `master` gets a version, and the affected branches another dedicated version.
I don't see a better solution for that case. :thinking:

Reply:
Integrated